parser/tools/generate_ast.cc

```diff
@@ -621,6 +621,12 @@ NodeDef nodes[] = {
         "rational",
         vector<FieldDef>({{"val", FieldType::String}}),
     },
+    // RBS placeholder for a bind comment
+    {
+        "RBSPlaceholder",
+        "rbs_placeholder",
+        vector<FieldDef>({{"kind", FieldType::Name}}),
```

Comment:
Another option would be to use `FieldType::Uint` here, and cast the values to an enum, but I think that using names is also fine.

Reply:
Avoiding the extra name, mainly. I'm happy to leave this for a follow-up 👍